plugins/hydra_ray_launcher/hydra_plugins/hydra_ray_launcher/conf/hydra/launcher/ray_aws.yaml

```diff
@@ -0,0 +1,159 @@
+hydra:
+  launcher:
+    cls: hydra_plugins.hydra_ray_launcher.ray_aws_launcher.RayAWSLauncher
+    params:
+      stop_cluster: True
+      # ray.init(**ray_init_config)
+      ray_init_cfg:
+        address: auto
+      # @ray.remote(**ray_remote_config)
+      ray_remote_cfg:
+        num_cpus: 1
+        num_gpus: 0
+      # this maps to a Ray cluster config, e.g:
+      # https://github.com/ray-project/ray/blob/master/python/ray/autoscaler/aws/example-full.yaml
+      # Note: You cannot add arbitrary field not supported by Ray
```

Comment:
I don't think we need to write this. anyone trying it will get an error when they do (and why would they do it anyway)?
```suggestion
```